packages/trpc/server/routers/viewer/bookings/get.handler.ts

```diff
@@ -185,7 +185,6 @@ export async function getBookings({
           select: {
             id: true,
             name: true,
```

Comment:
Shouldn't something like these would achieve a similar effect without the added complexity?
```suggestion
            name: true,
            members: {
              where: {
                userId: user.id,
                role: {
                  in: [MembershipRole.OWNER, MembershipRole.ADMIN],
                },
                select: {
                  role: true,
                },
              },
            },
```